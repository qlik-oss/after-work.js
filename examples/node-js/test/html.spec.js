import virtual from './foobar-virtual.html'; // eslint-disable-line
import template from './foobar.html';

describe('html', () => {
  it('should import virtual template', () => {
    expect(virtual).toMatchSnapshot();
  });

  it('should import template', () => {
    expect(template).toMatchSnapshot();
  });
});
