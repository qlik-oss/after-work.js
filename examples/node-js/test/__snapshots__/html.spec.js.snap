// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'should import template' 1`] = `"<div>foobar</div>"`;

exports[`'should import virtual template' 1`] = `"<div>hello world</div>"`;
